packages/config/src/forkConfig/index.ts

```diff
@@ -139,7 +166,7 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
       return sszTypesFor(forkName);
     },
     getMaxBlobsPerBlock(epoch: Epoch): number {
-      const fork = this.getForkInfoAtEpoch(epoch).name;
+      const {fork} = this.getForkBoundaryAtEpoch(epoch);
```

Comment:
It's the same as
`const fork = this.getForkInfoAtEpoch(epoch).name;` no?

Reply:
No, there is no difference, I reverted back to original code to reduce diff